docs/content/getting-started/setup-openfga.mdx

```diff
@@ -65,6 +65,12 @@ If using **Pre-shared key authentication**, you will configure OpenFGA with a se
 If you are going to use this setup in production, you should enable TLS in your OpenFGA server.
 :::
 
+### Playground
+
+The Playground facilitates rapid development by allowing you to visualize and model your application's authorization model(s) and manage relationship tuples with a locally running OpenFGA instance.
+
+The Playground is enabled on port 3000 by default. To disable the playground, please take a look at [Configuring the Server](#configuring-the-server) for more information.
```

Comment:
"To disable the Playground,"

Playground should be capitalized here and anywhere else